### PR TITLE
Final testing

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
Use jest framework

Create a test folder for a file(s) containing the following functions:
  a function for editing the task description.
  a function for updating an item's 'completed' status.
  the "Clear all completed" function.

### CODE REVIEW

**Comment by Ugboaja-Uchechi** [comment]
All linters have passed.

File: src/_tests_/edit.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import localStorageMock from '../_mocks_/localStorage.js';
+import editItem from '../_mocks_/edit.js';
```
Comment (rukundoeric) [inline src/_tests_/edit.test.js:6]: - [x] <b>REQUIRED:</b> Here, you are supposed to test the real functions with mocked data.  which means that the `editItem` function should be imported from `script2.js`. and then use mocked data. Check out this [Article](https://bholmes.dev/blog/mocking-browser-apis-fetch-localstorage-dates-the-easy-way-with-jest/) if you wanted to learn more on how to mock data in localStorage.

File: src/_tests_/remove.test.js [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import localStorageMock from '../_mocks_/localStorage.js';
 import removeItem from '../_mocks_/remove.js';
```
Comment (rukundoeric) [inline src/_tests_/remove.test.js:6]: - [x] <b>REQUIRED:</b> The same here, you are supposed to test the real functions with mocked data.  which means that the `removeItem` function should be imported from `script2.js`. and then use mocked data. Check out this [Article](https://bholmes.dev/blog/mocking-browser-apis-fetch-localstorage-dates-the-easy-way-with-jest/) if you wanted to learn more on how to mock data in localStorage.

File: src/_tests_/status.test.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import localStorageMock from '../_mocks_/localStorage.js';
+import status from '../_mocks_/status.js';
```
Comment (rukundoeric) [inline src/_tests_/status.test.js:6]: - [x] <b>REQUIRED:</b> The same here, you are supposed to test the real functions with mocked data.  which means that the `changeStatus` function should be imported from `script2.js`. and then use mocked data. Check out this [Article](https://bholmes.dev/blog/mocking-browser-apis-fetch-localstorage-dates-the-easy-way-with-jest/) if you wanted to learn more on how to mock data in localStorage.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Kingstalux, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/_tests_/edit.test.js (hunk 1 above)
Comment (Ugboaja-Uchechi) [reply src/_tests_/edit.test.js:6]: HI, my partner and I are a bit lost on the changes you requested, if you don't mind, can you join us on a zoom call

**Comment by Ugboaja-Uchechi** [comment]
My partners and I are testing the real functions but we copied those functions into individual files since we have to use mock data.
This is the same thing we did for the first testing project and it was approved.

**Review by AshakaE — APPROVED** [review]
### Hi @Ugboaja-Uchechi  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**